src/js/pages/universe/PackagesTab.js

```diff
@@ -109,9 +110,9 @@ class PackagesTab extends mixin(StoreMixin) {
     const {errorMessage} = this.state;
 
     return (
-      <CosmosErrorMessage
-        error={errorMessage}
-        headerClass="h3 text-align-center flush-top" />
+      <AlertPanel title="An Error Occurred">
+        <CosmosErrorMessage error={errorMessage} />
```

Comment:
I think we should add `flushBottom={true}` here, what do you think?

Without `flushBottom`:
![](https://cl.ly/392r372R2I19/Screen%20Shot%202017-03-02%20at%2011.38.02%20AM.png)

With `flushBottom`:
![](https://cl.ly/0P3B2X321L3l/Screen%20Shot%202017-03-02%20at%2011.38.21%20AM.png)